cmd/create_data_stream.go

```diff
@@ -10,13 +10,12 @@ import (
 	"maps"
 	"slices"
 
-	"github.com/AlecAivazis/survey/v2"
-
 	"github.com/spf13/cobra"
 
 	"github.com/elastic/elastic-package/internal/packages"
 	"github.com/elastic/elastic-package/internal/packages/archetype"
 	"github.com/elastic/elastic-package/internal/surveyext"
```

Comment:
Nit. Now that we have an internal package to provide the `survey` functionality we don't need this `...ext` package, this could be merged into the `tui internal package.